website/events/templates/events/event.html

```diff
@@ -44,6 +44,24 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                 <div class="col-12 col-md-6 order-1">
                     <table class="table table-borderless">
                         <tbody>
+                        {% if event.album_set.all %}
```

Comment:
Could this button be moved down to the end of the event information?

Reply:
I actually like it here because this will only appear after the event has ended so the rest of the event information is less important then.

Reply:
It's still the information of the event, the link to the photo album is secondary in my opinion since it does not describe the event in any way.

Reply:
I also think it's better to place the button below the event information.

Reply:
Okay... clear enough I'm in the minority. I have put it all the way down now.